differ/differ.go

```diff
@@ -539,14 +539,16 @@ func processReportsByCluster(config conf.ConfigStruct, ruleContent types.RulesMa
 			produceEntriesToServiceLog(config, cluster, ruleContent, deserialized.Reports)
 		}
 
-		newNotifiedIssues, err := produceEntriesToKafka(cluster, ruleContent, deserialized.Reports, storage, report)
-		if err != nil {
-			log.Err(err).
-				Str(clusterAttribute, string(cluster.ClusterName)).
-				Msg("Unable to send the notification message to Kafka")
-			continue
+		if conf.GetKafkaBrokerConfiguration(config).Enabled {
```

Comment:
or `if !conf.GetKafkaBrokerConfiguration(config).Enabled {continue}`

Reply:
that's the way to Go =)